README.rst

```diff
@@ -0,0 +1,315 @@
+========================================================
+REANA example - LHCb Rare Charm Decay Search
```

Comment:
In the mean heading, there should be exactly one leading and one trailing space, like this:

```
=====
 Foo 
=====
```